cadc-vosi/src/main/java/ca/nrc/cadc/vosi/AvailabilityServlet.java

```diff
@@ -21,74 +24,70 @@
 /**
  * Servlet implementation class CapabilityServlet
  */
-public class AvailabilityServlet extends HttpServlet
-{
+public class AvailabilityServlet extends HttpServlet {
     private static Logger log = Logger.getLogger(AvailabilityServlet.class);
     private static final long serialVersionUID = 201003131300L;
 
     private String wsClassName;
-    
+
     @Override
     public void init(ServletConfig config)
-        throws ServletException
-    {
+        throws ServletException {
         this.wsClassName = config.getInitParameter("ca.nrc.cadc.vosi.WebService");
         log.info("WebService class name: " + wsClassName);
     }
 
     @Override
     protected void doGet(HttpServletRequest request, HttpServletResponse response)
-        throws ServletException, IOException
-    {
+        throws ServletException, IOException {
         boolean started = false;
         WebServiceLogInfo logInfo = new ServletLogInfo(request);
         long start = System.currentTimeMillis();
-        try
-        {
+        try {
             Subject subject = AuthenticationUtil.getSubject(request);
             logInfo.setSubject(subject);
             log.info(logInfo.start());
-            
+
             Class wsClass = Class.forName(wsClassName);
             WebService ws = (WebService) wsClass.newInstance();
 
             AvailabilityStatus status = ws.getStatus();
+
+            // TODO: Setting the client IP here may be destructive as it overrides whatever was (potentially) set in the
+            // TODO: WebService instance's constructor.
+            //
+            // TODO: jenkinsd 2018.02.06
+            //
+            status.setClientIP(NetUtil.getClientIP(request));
+
```

Comment:
See comment in AvailabilityStatus.java -- this new section of code can be removed